google_calendar_clickable_where/clickable_where.user.js

```diff
@@ -57,34 +29,25 @@ w.clickable_where.make_all_links_clickable = function make_all_links_clickable (
                     g = n.textContent.match(exp);
                 }
             }
-            else if (n.tagName.toLowerCase() != 'a') {
-                w.clickable_where.make_all_links_clickable(n);
+            else if ((n.nodeType == n.ELEMENT_NODE) && (n.tagName.toLowerCase() != 'a')) {
+                window.make_child_links_clickable(n);
             }
         }
     }
 };
 
-if (typeof (opera) != "undefined") {
-    // Start the mainpart, if the site is loaded
-    window.addEventListener('DOMContentLoaded', w.clickable_where.main, true);
-} else {
-    w.clickable_where.main();
+function whereClickHandler(e) {
+    //console.log("Click Handler Fired");
+    // check for pop-up meeting info modal
+    nodes = window.document.querySelectorAll('.neb-break-words');
+    for (var i = 0, m = nodes.length; i < m; i++) {
+        window.make_child_links_clickable(nodes[i]);
+    }
+    // check for event detail
+    nodes = window.document.querySelectorAll('.ui-sch-schmedit');
+    for (var i = 0, m = nodes.length; i < m; i++) {
+        window.make_child_links_clickable(nodes[i]);
+    }
 }
 
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-// Fix for user scripts not working in opera unless lots of end content exists at the end of the file.  This text is useful only in that it is very long and fills up a lot of space, ensuring that Opera will be able to load this user script.
+window.document.addEventListener('click', whereClickHandler);
```

Comment:
This will not work in most versions of Opera. The check for version was needed to ensure this userscript will work if added directly to Opera as well.